packages/babel-parser/src/parse-error/standard-errors.js

```diff
@@ -129,6 +129,9 @@ export default (_: typeof toParseErrorCredentials) => ({
   ),
   ImportCallNotNewExpression: _("Cannot use new with import(...)."),
   ImportCallSpreadArgument: _("`...` is not allowed in `import()`."),
+  ImportJSONBindingNotDefault: _(
+    `A JSON module can only be imported with "default".`,
```

Comment:
```suggestion
    `JSON modules only expose the default import.`,
```

Having `"default"` in quotes makes it looks like it's a string :sweat_smile:

Reply:
I personally prefer `can only be imported` because it's more of an error than a warning.😃

Reply:
Maybe we can use:
"A JSON module can only be imported with `default`."
Like " `import()` " above.

Reply:
```suggestion
    "A JSON module can only be imported with `default`.",
```

Reply:
> Having "default" in quotes makes it looks like it's a string

Technically we can, I deliberately put `import { "default" as foo } ...` in one of tests.